src/models/spaces/populations/mouse-model/mouse-populations-model.ts

```diff
@@ -69,56 +72,82 @@ export const MousePopulationsModel = types
           }
         },
 
+        get environmentColor(): EnvironmentColor {
```

Comment:
True. Done. I had originally thought it best to do it that way because the authoring object was being passed in directly, and authors may change it by hand and make a typo, but I realized it's much better to have MST do the type-checking and fail early in that scenario.